projects/1.18/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -123,240 +137,519 @@
   "item.gobber2.gobber2_chestplate_end": "末影胸甲",
   "item.gobber2.gobber2_leggings_end": "末影护腿",
   "item.gobber2.gobber2_boots_end": "末影靴子",
-  "item.gobber2.gobber2_armor_dragon.line1": "拥有末影套装的所有能力",
-  "item.gobber2.gobber2_armor_dragon.line2": "满装加成：永久飞行和消除负面影响",
-  "item.gobber2.gobber2_armor_dragon.line3": "满装加成：负面效果抗性",
+
+  "item.gobber2.gobber2_armor_dragon.line1":"套装效果：",
+  "item.gobber2.gobber2_armor_dragon.line2":"获得饱和效果",
+  "item.gobber2.gobber2_armor_dragon.line3":"免疫所有负面状态影响",
+  "item.gobber2.gobber2_armor_dragon.line4":"拥有末影套装的所有能力",
+  "item.gobber2.gobber2_armor_dragon.line5":"得到所有维度的永久创造飞行能力",
   "item.gobber2.gobber2_helmet_dragon": "龙之头盔",
   "item.gobber2.gobber2_chestplate_dragon": "龙之胸甲",
   "item.gobber2.gobber2_leggings_dragon": "龙之护腿",
   "item.gobber2.gobber2_boots_dragon": "龙之靴子",
-  "item.gobber2.gobber2_links_end": "末影装甲锁链",
-  "item.gobber2.gobber2_ring.cooldown": "冷却时间：%s",
-  "item.gobber2.gobber2_ring": "戈伯戒指",
-  "item.gobber2.gobber2_ring_nether": "下界戒指",
-  "item.gobber2.gobber2_ring_end": "末影戒指",
-  "item.gobber2.gobber2_ring_return": "回城戒指",
-  "item.gobber2.ring_return.line1": "欢迎回家！！",
-  "item.gobber2.ring_return.line2": "必须先右击床设置重生点",
-  "item.gobber2.ring_return.line3": "长按右击返回重生点",
-  "item.gobber2.ring_return.line4": "只能在主世界使用",
+
+  "item.gobber2.gobber2_links_end":"末影装甲锁链",
+  "item.gobber2.sneak_right_click":"潜行单击右键以打开/关闭",
+  "item.gobber2.works_in_enderchest":"在Endercest工作",
+  "item.gobber2.works_in_inventory":"库存时工作",
+  "item.gobber2.works_in_main_hand":"主控时有效",
+  "item.gobber2.works_in_off_hand":"不在时工作",
+  "item.gobber2.right_click_use":"右键单击可使用",
+  "item.gobber2.gobber2_ring.cooldown":"冷却间隔(tick)： %s",
+
+  "item.gobber2.gobber2_ring":"戒指",
+  "item.gobber2.gobber2_ring_nether":"下界戈伯戒指",
+  "item.gobber2.gobber2_ring_end":"末地戈伯戒指",
+
+  "item.gobber2.gobber2_ring_above":"上（下）戒指",

```

```suggestion
  "item.gobber2.gobber2_ring_above":"上天（下地）戒指",
```
仅供参考，个人认为这个词组去除括号依旧得说得通